site/_docs/upgrading/2-to-3.md

```diff
@@ -99,4 +99,8 @@ This can be fixed by removing the following line from your `_config.yml` file:
 relative_permalinks: true
 {% endhighlight %}
 
+### Permalinks no longer automatically add a trailing slash
+
+In Jekyll 2, any URL constructed from the `permalink:` field had a trailing slash (`/`) added to it automatically.  Jekyll 3 no longer adds a trailing slash automatically to `permalink:` URLs. This can potentially result in old links to pages returning a 404 error. For example, suppose a page previously contained the YAML `permalink: /:year-:month-:day-:title` that resulted in the URL `example.com/2016-02-01-test/` (notice the trailing slash). In Jekyll 3, the URL for the same page is `example.com/2016-02-01-test`, and consequently any links to the old URL will result in a 404 error. In order to maintain the same URLs and avoid this problem, a trailing slash should be added to the `permanlink:` field, for example `permalink: /:year-:month-:day-:title/`. 
```

Comment:
There's a `permanlink` misspelling in here. Would you mind fixing that up really quick? Thank you! Then I'll merge :)